app/src/main/java/com/example/ohthmhyh/CustomAdapterHF.java

```diff
@@ -1,7 +1,9 @@
 package com.example.ohthmhyh;
```

Comment:
Can you add a test in HabitFragmentTest that tests creating a Habit, then creates HabitEvents associated with that Habit, and then deletes the Habit and ensures that the associated HabitEvents were deleted? Thanks!